app/uk/gov/hmrc/eusubsidycompliancefrontend/services/BusinessEntityJourney.scala

```diff
@@ -47,23 +47,31 @@ case class BusinessEntityJourney(
     )
 
   def setAddBusiness(b: Boolean): BusinessEntityJourney =
-    this.copy(
-      addBusiness = addBusiness.copy(value = b.some))
+    this.copy(addBusiness = addBusiness.copy(value = b.some))
 
 }
 
 object BusinessEntityJourney {
 
+  val eoriPrefix = "GB"
+
   implicit val format: Format[BusinessEntityJourney] = Json.format[BusinessEntityJourney]
 
+  def isEoriPrefixGB(eoriEntered: String) = eoriEntered.take(2) === eoriPrefix
+
+  def getValidEori(eoriEntered: String) =
+    if (isEoriPrefixGB(eoriEntered)) eoriEntered else s"$eoriPrefix$eoriEntered"
```

Comment:
I don't _think_ we ever want the user to add in the prefix. Is this to allow them to if they want?

Reply:
It is the new requirement, where user haven't seen the GB  on the form and entered the eORI with GB or copy pasted , then they shouldn't get the error message.

Reply:
ah ignore that - this is consistent with other place. 👍